core/server/data/validation/index.js

```diff
@@ -17,19 +17,19 @@ var schema    = require('../schema').tables,
 // Provide a few custom validators
 //
 validator.extend('empty', function empty(str) {
-    return _.isEmpty(str);
+    return _.isEmpty(str+'');
```

Comment:
This is failing because there aren't spaces here. Update it to be like this: `str + ''` (That goes for all of them) :smile: